modules/orchestrator/proc-enable-orchestrator-plugin.adoc

```diff
@@ -9,31 +9,27 @@ You can enable the Orchestrator plugin in {product-very-short} by configuring dy
 * You have access to edit or create ConfigMaps in the namespace where the {product-custom-resource-type} CR is deployed.
 
 .Procedure
-. Identify the Orchestrator plugin dynamic plugins that are included by default in the default `dynamic-plugins.yaml` file of the `install-dynamic-plugins` container:
-+
-* `@redhat/backstage-plugin-orchestrator` (frontend)
-* `@redhat/backstage-plugin-orchestrator-backend-dynamic` (backend)
-* `@redhat/backstage-plugin-scaffolder-backend-module-orchestrator-dynamic`
-* `@redhat/backstage-plugin-orchestrator-form-widgets`
 
-. To enable the Orchestrator plugin, update the ConfigMap file with the following data:
-+
-[source,yaml,subs="+attributes,+quotes"]
+. To enable the Orchestrator plugin with default settings, set `disabled: false` for the package. For example, `package: "@redhat/backstage-plugin-orchestrator@<plugin-version>` is set to `disabled: false`:
 +
+[source,yaml]
 ----
-    includes:
-       - dynamic-plugins.default.yaml
-    plugins:
-       - package: "@redhat/backstage-plugin-orchestrator@1.6.0"
-         disabled: false
-       - package: "@redhat/backstage-plugin-orchestrator-backend-dynamic@1.6.0"
-         disabled: false
-         dependencies:
-            - ref: sonataflow
-       - package: "@redhat/backstage-plugin-scaffolder-backend-module-orchestrator-dynamic@1.6.0"
-         disabled: false
-      - package: "@redhat/backstage-plugin-orchestrator-form-widgets@${ORCHESTRATOR_PLUGIN_VERSION}"
-         disabled: false
+- package: "@redhat/backstage-plugin-orchestrator@<plugin-version>"
+    disabled: false
```

Comment:
Just to enable with default settings package and disabled is enough, no needs in other fields (integrity, pluginConfig)